Cargo.toml

```diff
@@ -37,3 +37,4 @@ eyre = "0.6.5"
 quickcheck = "1"
 quickcheck_macros = "1"
 env_logger = "0.8"
+hex = "0.4"
```

Comment:
nit: missing new line

Reply:
I added .editorconfig to make sure all files follow the same pattern. More info [here](https://editorconfig.org/#file-format-details)

Reply:
Fixed.